src/platform-apps/channels/channel-view.test.tsx

```diff
@@ -116,4 +117,66 @@ describe('ChannelView', () => {
 
     expect(wrapper.find(Waypoint).exists()).toBe(false);
   });
+
+  describe('Lightbox', () => {
+    it('renders when image file is within message and LightBox has been opened', () => {
+      const imageMedia = { url: 'image.jpg', type: MediaType.Image };
+      const messages = [
+        {
+          id: 'message-1',
+          message: 'image message',
+          media: imageMedia,
+          sender: { userId: 1 },
+          createdAt: 1658776625730,
+        },
+        {
+          id: 'message-2',
+          message: 'video message',
+          media: { url: 'video.avi', type: MediaType.Video },
+          sender: { userId: 1 },
+          createdAt: 1658776625731,
+        },
+        {
+          id: 'message-3',
+          message: 'audio message',
+          media: { url: 'video.mp3', type: MediaType.Audio },
+          sender: { userId: 1 },
+          createdAt: 1658776625732,
+        },
+      ];
+      const wrapper = subject({ messages });
+      wrapper.instance().openLightbox();
```

Comment:
let's not call internal methods. we should be able to trigger this from the boundary so that we're not depending on implementation details. something like:
```
wrapper.find(Message).at(1).simulate('imageClicked');
```